indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/Acl.java

```diff
@@ -425,6 +467,11 @@ public static Principal getUserPrincipal(String userId, String identitySourceId)
    */
   public static Principal getGroupPrincipal(String groupId) {
     checkArgument(!Strings.isNullOrEmpty(groupId), "Group ID can not be empty or null");
+    if (Configuration.isInitialized()
```

Comment:
Add a comment about what this is for, and a TODO to see whether the contains being case-sensitive is a problem.